src/data/sources.json

```diff
@@ -42,7 +42,7 @@
   {
     "name": "CNN",
     "site": "https://edition.cnn.com/business",
-    "base": ""
+    "base": "https://www.cnn.com"
```

Comment:
I'm interested to know if this worked when you tested it? I can see that they have automatic forwarding when you access www.cnn.com in the browser. I wonder if it still works with the addition of a URI?

But as far as I know when I did, the base needs to be:
```
"https://edition.cnn.com"
```

Reply:
I did test it, and it worked. But you're right it does redirect to https://edition.cnn.com/ I will change it.